environments/jvm/README.md

```diff
@@ -0,0 +1,64 @@
+# Fission: JVM Environment
```

Comment:
Can we say "Java and JVM" environment?  For people searching specifically for "Java"